src/Orleans.EventSourcing/LogStorage/LogConsistencyProvider.cs

```diff
@@ -16,12 +13,8 @@ namespace Orleans.EventSourcing.LogStorage
     /// metadata (the log position, and write flags) are stored in the primary. 
     /// </para>
     /// </summary>
-    public class LogConsistencyProvider : ILogConsistencyProvider
```

Comment:
Oh I see.... they are all called LogConsistentcyProvider, which is why all those extension methods looks like doing the same thing.... Can you change their name to be more specific?

Reply:
There are 2 other pending change sets for event sourcing.  Refactoring this should be done either when addressing those, or after they go in.  While I understand how the current patterns may not be clear to maintainers (us), these ambiguities do not hinder the end user experience.